src/test/buf/buf.test2.yaml

```diff
@@ -0,0 +1,26 @@
+# Below is from https://buf.build/docs/configuration/v1/buf-yaml.
+
+version: v1
```

Comment:
These tests definitely aren't stress-tests, more like sanity-checks - happy to extend them.

Reply:
It's up to you - usually I just copy and paste a bunch of snippets from the documentation and call it a day - which doesn't take too long. But if you prefer to only have the sanity checks, we'll definitely accept that too!

Reply:
yeah, I think for a first-go these are fine. If I missed something I'll definitely cover it in a test.